src/ui/modals/confirm_box.rs

```diff
@@ -0,0 +1,77 @@
+use ::tui::buffer::Buffer;
+use ::tui::layout::Rect;
+use ::tui::style::{Color, Modifier, Style};
+use ::tui::widgets::Widget;
+
+use crate::ui::format::truncate_middle;
+use crate::ui::grid::draw_filled_rect;
+
+fn render_confirm_prompt(buf: &mut Buffer, confirm_rect: &Rect, confirm_msg: &str) {
+    let text_style = Style::default()
+        .bg(Color::Black)
+        .fg(Color::White)
+        .modifier(Modifier::BOLD);
+
+    let text_max_length = confirm_rect.width - 4;
+    let confirm_text = truncate_middle(confirm_msg, text_max_length);
+    let confirm_text_start_position =
+        ((confirm_rect.width - confirm_text.len() as u16) as f64 / 2.0).ceil() as u16
+            + confirm_rect.x;
+    let y_n_line = "(y/n)";
+    let y_n_line_start_position =
+        ((confirm_rect.width - y_n_line.len() as u16) as f64 / 2.0).ceil() as u16 + confirm_rect.x;
+
+    buf.set_string(
+        confirm_text_start_position,
+        confirm_rect.y + confirm_rect.height / 2 - 2,
+        confirm_text,
+        text_style,
+    );
+    buf.set_string(
+        y_n_line_start_position,
+        confirm_rect.y + confirm_rect.height / 2,
```

Comment:
Can we lower the y/n line a little so the message is more spread out on the window? I think it'll look a little better. What do you think?

Reply:
Of course! thanks for the remark. To be honest, it wasn't a good idea in the first place now that i think of it.
I think it might be good if i add a "+ 3" at the end of the line: 
`confirm_rect.y + confirm_rect.height / 2 + 3,`
(this is the exact position of the y/n line in deletion_prompt.)